CMakeLists.txt

```diff
@@ -898,6 +916,9 @@ if(NRN_ENABLE_MPI)
     message(STATUS "  LIB         | ${MPI_LIBRARY}")
   endif()
 endif()
+if(NRN_ENABLE_MUSIC)
+  message(STATUS "MUSIC         | ${MUSIC_PREFIX}")
```

Comment:
Is FindMusic setting MUSIC_PREFIX? If not we could show MUSIC_INCDIR here.

Reply:
Yes, I need to update this to fit the changed FindMusic.cmake. Thanks for the heads-up.